extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -108,58 +124,12 @@ class AmpInstagram extends AMP.BaseElement {
         setStyles(iframe, {
           'opacity': 1,
         });
-
-        // Hide the initial rendered image to avoid overlaying videos.
-        setStyles(this.placeholderWrapper_, {
-          'display': 'none',
-        });
+        this.togglePlaceholder(false);
```

Comment:
This will not work unless you will also override `firstLayoutCompleted`. The default implementation hides placeholder for you. So, if you want it to stick around, you have to override. But, to be honest, I don't really see much point to it. `vsync` will be very fast, so you can just forgo toggling placeholder here entirely.